src/web/server/locales/en/common.json

```diff
@@ -132,7 +132,7 @@
   },
   "decisionFallbackSuccessful": {
```

Comment:
presumably this will be deleted before the story is complete?

Reply:
Yes, that will be the next PR, all fallback code will be removed.